File: app/src/main/java/doit/study/droid/data/QuizDataClient.java
package doit.study.droid.data;

import java.util.List;

import retrofit2.Call;
import retrofit2.http.GET;

public interface QuizDataClient {
    @GET("quiz.json?cache_stub=drop")
    Call<List<Question>> getQuestions();
    @GET("config?cache_stub=drop")
    Call<Configuration> getConfiguration();
}

File: app/src/main/java/doit/study/droid/data/QuizDataClient.kt
package doit.study.droid.data

import retrofit2.Call
import retrofit2.http.GET

interface QuizDataClient {
    @GET("quiz.json?cache_stub=drop")
    suspend fun questions(): List<Question>
    @GET("config?cache_stub=drop")
    suspend fun configuration(): Configuration
}
